examples/braket_features/Getting_Started_with_OpenQASM_on_Braket.ipynb

```diff
@@ -138,15 +138,16 @@
    "source": [
```

Comment:
<p>I feel this title is redundant and unnecessary "OpenQASM features on Braket"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>This is an already existing notebook. I only changed one cell to show that you can use the LocalSimulator. The rest of the notebook uses SV1 still.</p>